creativecloud/features/genfill/genfill-interactive.js

```diff
@@ -0,0 +1,99 @@
+import { getLibs } from '../../scripts/utils.js';
+import { createEnticement } from '../interactive-elements/interactive-elements.js';
+import defineDeviceByScreenSize from '../../scripts/decorate.js';
+
+const miloLibs = getLibs();
+const { createTag } = await import(`${miloLibs}/utils/utils.js`);
+
+function handleTransition(pics, index) {
+  pics[index].style.display = 'none';
+  const nextIndex = (index + 1) % pics.length;
+  pics[nextIndex].style.display = 'block';
+  return nextIndex;
+}
+
+function startAutocycle(interval, pics, clickConfig) {
+  if (clickConfig.isImageClicked) return;
+  clickConfig.autocycleInterval = setInterval(() => {
+    clickConfig.autocycleIndex = handleTransition(pics, clickConfig.autocycleIndex);
+    if (clickConfig.autocycleIndex === pics.length - 1) {
+      clearInterval(clickConfig.autocycleInterval);
+    }
+  }, interval);
+}
+
+function handleClick(aTags, clickConfig) {
+  aTags[0].style.display = 'block';
```

Comment:
We already have 0th element set to block in CSS. We do not need to reset it again